src/containers/DentistCreateForm/DentistCreateForm.container.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { dentistCreateAccount } from "../../services/teeth-apis/RegisterController";
+import { notification } from "antd";
+
+import DentistCreateFormComponent from "../../components/forms/DentistCreateFormComponent/DentistCreateForm.component";
+import DentistFormValueToDentistData from "../../mapper/DentistFormValueToDentistData";
+
+const DentistCreateFormContainer = () => {
```

Comment:
is this container worth ? actually you only need to put all in the CreateStaffModal Container and then check which tab is using to call the right API tbh